app/controllers/api/articles_controller.rb

```diff
@@ -4,13 +4,30 @@ class Api::ArticlesController < ApplicationController
   before_action :authenticate_user!, only: [:create]
 
   def index
-    article = Article.where(published: true)
-    render json: article, each_serializer: Article::IndexSerializer
+    check_index_params
+    category = params[:category] || 'all'
+
+    case category
+    when 'local'
+      articles = find_articles({}, { id: -1 })
```

Comment:
why is the id -1?

Reply:
Because we want *only* local news, we don't want to include international. So the database query `.or(Article.where(international: true, id: -1)` will never match any results, since no article can have id: -1.